### ISSUE TITLE
Summary Buff- and Aura-Stacking

### ISSUE THREAD

**Issue by drey113** [tobmaps/TrinityCore#195]
There are lots auras which are stacking. Players make huge dps, almost like in cata ^_^. It would be nice, if u get a look.
https://github.com/TrinityCore/TrinityCore/issues/1225

**Comment by Wattiqelumes** [comment]
Seriously not a single one of these expected. They are ALL fixed in this repo long ago. =P

And yeah, I'm serious, every single one of them is fixed. D:

**Comment by drey113** [comment]
i see now, but when will be fixes on https://github.com/TrinityCore/TrinityCore/commits/ ? our server admin, use only TC for updates ;S

**Comment by Wattiqelumes** [comment]
Well, your server owner is a bitch. I bet he takes credits for all these fixes as well..

**Comment by drey113** [comment]
I don't think so, anyway it sucks, because he don't give a f**\* about server, just time to time update using TC.
